### PR TITLE
[strings] Added italian translations

### PR DESCRIPTION
**Pull request by FinixFighter**
Signed-off-by: Luca Finizio <luca.finizio@protonmail.com>

### CODE REVIEW

File: data/strings/types_strings.txt [hunk 1]
```diff
@@ -5015,7 +5020,7 @@
     fi = Ilmastointilaite
     fr = Climatiseur
     hu = Hűtés-fűtés szerelő
-    it = Impianti HVAC
+    it = Impianti HVAC | Impianti di climatizzazione
```
Comment (biodranik) [inline data/strings/types_strings.txt:5023]: Alternate translations are not allowed in types strings

**Review by biodranik — CHANGES_REQUESTED** [review]

**Comment by FinixFighter** [comment]
The "Merge branch 'organicmaps:master' into master" can be skipped!

File: data/strings/types_strings.txt [hunk 2]
```diff
@@ -4842,12 +4843,14 @@
...
   [type.craft.beekeeper]
     en = Beekeeper
+    it = Apicoltore
     be = Пчаляр
     ru = Пчеловод
     uk = Бджоляр
 
   [type.craft.blacksmith]
     en = Blacksmith
+    it = Fabbro
```
Comment (biodranik) [inline data/strings/types_strings.txt:4853]: Sort?

File: data/strings/types_strings.txt [hunk 3]
```diff
@@ -4842,12 +4843,14 @@
 
   [type.craft.beekeeper]
     en = Beekeeper
+    it = Apicoltore
```
[the remark this answers is not in the document]
Comment (FinixFighter) [reply data/strings/types_strings.txt:4846]: > Sort?

Sorry, I don't understand what you mean with "sort".

[the remark this answers is not in the document]
Comment (biodranik) [reply data/strings/types_strings.txt:4846]: It means:
- en always goes first
- All other languages are sorted in the English alphabet order.

**Pull request closed without merging**